spec.md

```diff
@@ -95,61 +105,92 @@ Everything that applies to licenses in this section also applies to license
+and Licensing Information in a comment header. The Information SHOULD be as
+close to the top of the top of the file as possible in a comment header. The
+Commentable File SHOULD use UTF-8 encoding.
 
 For Uncommentable Files, the comment header that declares the file's Copyright
-and Licensing Information SHOULD be in an adjacent UTF-8 encoded text file of the same 
-name with the additional extension `.license` (example: `cat.jpg.license` if the 
-original file is `cat.jpg`).
+and Licensing Information MUST be in an adjacent text file of the same name with
+the additional extension `.license` (example: `cat.jpg.license` if the original
+file is `cat.jpg`). The adjacent file SHOULD use UTF-8 encoding.
 
 `.license` files MAY be used with Commentable Files, but it is still RECOMMENDED
 that comment headers be put inside Commentable Files.
 
-The comment header MUST contain one or more `SPDX-FileCopyrightText` tags, and one or
-more `SPDX-License-Identifier` tags. A tag is followed by a colon, followed by
-a text value, and terminated by a newline.
-
-The `SPDX-FileCopyrightText` tag MUST be followed by a copyright notice.
-
-Instead of the `SPDX-FileCopyrightText` tag, the symbol `©`, or the word `Copyright` MAY
-be used, in which case a colon is not needed.
+The comment header MUST contain one or more Copyright Notices and one or more
+`SPDX-License-Identifier` tag-value pairs. A tag is followed by a colon,
+followed by a text value, and terminated by a newline.
 
 The `SPDX-License-Identifier` tag MUST be followed by a valid SPDX License
 Expression describing the licensing of the file (example:
-`SPDX-License-Identifier: GPL-3.0-or-later OR Apache-2.0`). If separate sections
-of the file are licensed differently, a different `SPDX-License-Identifier` tag
-MUST be included for each section.
+`SPDX-License-Identifier: GPL-3.0-or-later OR Apache-2.0`).
 
 An example of a comment header:
 
 ```
 # SPDX-FileCopyrightText: 2016, 2018-2019 Jane Doe <jane@example.com>
-# SPDX-FileCopyrightText: 2019 Example Company
+# SPDX-FileCopyrightText: 2019 Example NGO
 #
 # SPDX-License-Identifier: GPL-3.0-or-later
 ```
 
-If these tags are additionally used in the file without describing the file's
-actual license or copyright but for example as part of an output command or
-documentation, these occurrences MAY be put between two comments:
-`REUSE-IgnoreStart` and `REUSE-IgnoreEnd`. The REUSE Tool then ignores all tags
-within. This technique MUST NOT be used to ignore valid tags for licensing or
-copyright.
+#### In-line Snippet comments
+
+If some Copyright and Licensing Information inside of a Commentable File is to
+apply only to a certain Snippet instead of the whole file, SPDX snippet tags
+MUST be used for that Snippet (as defined in [SPDX Specification, Annex
+H](https://spdx.github.io/spdx-spec/v2.3/file-tags/#h3-snippet-tags-format)).
+This means that Copyright Notices inside of Snippets MUST be prefixed with
+`SPDX-SnippetCopyrightText`.
```

Comment:
Attempt to use more simple language:
```suggestion
Sometimes Copyright and Licensing Information should only apply to a certain
Snippet instead of the whole Commentable File. For these cases SPDX snippet tags
MUST be used for that Snippet (as defined in [SPDX Specification, Annex
H](https://spdx.github.io/spdx-spec/v2.3/file-tags/#h3-snippet-tags-format)).
This means that Copyright Notices inside of Snippets MUST be prefixed with
`SPDX-SnippetCopyrightText`.
```

Reply:
Ack, slightly modified ('should only apply' → 'only applies')